### PR TITLE
Remove //norelease and cleans up some aggregations tests

### PR DESCRIPTION
**Pull request by tlrx**
Note: PR against a feature branch

This commit removes some //norelease tags and cleans up remaining @Before.

### CODE REVIEW

**Review by cbuescher — APPROVED** [review]
@tlrx thanks for doing this, LGTM

**Comment by tlrx** [comment]
Thanks @cbuescher

File: core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java [hunk 1]
```diff
@@ -27,16 +27,16 @@
...
 
 public class InternalExtendedStatsTests extends InternalAggregationTestCase<InternalExtendedStats> {
-    protected double sigma;
 
-    @Before
-    public void randomSigma() {
+    private double sigma;
+
+    @Override
+    public void setUp() throws Exception {
```
Comment (javanna) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java]: I wonder how this one happened to work in AggregationsTests :)

Comment (tlrx) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java]: looking right now

Comment (tlrx) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java]: AggregationsTests tests it using the default double value 0.0 for `sigma` which produces a valid XContent output that can be parsed back.

Comment (javanna) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStatsTests.java]: right it's a double not Double, then no NPE :) makes sense

**Pull request merged**